WordPress/Classes/ViewRelated/Views/WPAnimatedBox.m

```diff
@@ -73,8 +73,6 @@ - (void)prepareAnimation:(BOOL)animated
                              view.transform = CGAffineTransformMakeTranslation(0, CGRectGetHeight(self.frame) - YOrigin);
                          }
                      }];
-
-    _isPreparedToAnimate = YES;
```

Comment:
With this removed, `_isPreparedToAnimated` is never set to `YES`... so can we just remove it entirely?